### ISSUE TITLE
Using bun in monorepo with yarn workspaces fails to install dependencies

### ISSUE THREAD

**Issue by caiocampoos** [oven-sh/bun#5274]
### What version of Bun is running?

1.0.1

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

bun install 

### What is the expected behavior?

should isntall dependencies for a monorepo with yarn workspaces config

### What do you see instead?

error @company/package not found
error @company/package2 not found
error @company/package3 not found
error @company/package4 not found

**Comment by BhavyaCodes** [comment]
Im having the same issue

```
bun install
bun install v1.0.1 (31aec4eb)
  🔍 @fibonacci-calculator/server [1/1] 
error: package "@fibonacci-calculator/server" not found registry.npmjs.org/@fibonacci-calculator/server 404
error: @fibonacci-calculator/server@* failed to resolve
```

**Comment by caiocampoos** [comment]
We found out that deleting the lockfile make it work, but then i have to delete it every time.

**Comment by ftognetto** [comment]
The solution is in this pr! https://github.com/oven-sh/bun/pull/5379

**Comment by Electroid** [comment]
Closed by #5379. If you're still experiencing issues with `bun install` and monorepos, let us know in a new issue and we'll investigate.
